modules/gdscript/gdscript_parser.cpp

```diff
@@ -3422,7 +3422,16 @@ void GDScriptParser::get_class_doc_comment(int p_line, String &p_brief, String &
 				p_tutorials.append(Pair<String, String>(title, link));
 				break;
 			case DONE:
-				return;
+				break;
+		}
+	}
+	if (current_class->members.size() > 0) {
+		const ClassNode::Member &m = current_class->members[0];
+		int first_member_line = m.get_line();
+		if (first_member_line == line) {
```

Comment:
After `p_brief` and `p_desc` and `p_tutorials` are populated, `line` points to the next line. I compare it with the line of the first member of the class and if it's the same line - then discard the values populated because they belong to the member.